diff --git a/Package.swift b/Package.swift
--- a/Package.swift
+++ b/Package.swift
@@ -6,7 +6,7 @@ let package = Package(
     name: "TreeSitterSwift",
     platforms: [.macOS(.v10_13), .iOS(.v11)],
     products: [
-        .library(name: "TreeSitterSwift", targets: ["TreeSitterSwift"]),
+        .library(name: "TreeSitterSwift", type: .static, targets: ["TreeSitterSwift"]),
     ],
     dependencies: [],
     targets: [
